week-9/Homework/mandatory/2-exercises/2-convertion.js

```diff
@@ -1,30 +1,34 @@
-/*
-    Convert the Function here into a JavaScript Class called 'Person'
-
-    ```
-        function createNewPerson(name) {
-            const obj = {};
-            obj.name = name;
-            obj.greeting = function() {
-                alert('Hi! I\'m ' + obj.name + '.');
+
+    //Convert the Function here into a JavaScript Class called 'Person'
+
+
+       // function createNewPerson(name) {
+class Person{
+    constructor(name) {
+        
```

Comment:
👍 - Have you got a code formatter to help you out with the formatting of your code and the indentation ?

Reply:
Not untill now but thank you for your comment I downloaded it now.